packages/styles/scss/components/controls/_chat-toggle.scss

```diff
@@ -2,5 +2,16 @@
 
 .chat-toggle {
   @extend .button;
-  position: initial; // Only here to include this selector into the type gen process.
+  position: relative;
+}
+
+.chat-toggle[data-unread='true']:after {
+  content: '';
+  position: absolute;
+  top: 0;
+  right: 0;
+  transform: translate(50%, -50%);
+  background: var(--lk-accent-bg);
+  padding: 0.375rem;
+  border-radius: 50%;
```

Comment:
<img width="383" alt="image" src="https://github.com/livekit/components-js/assets/11357413/dfe67918-1f7c-40a4-84f1-40b336d3c2fd">

The indicator looks nice in your UI. I'm a little concerned, though, when the default position goes over the edge of the button. This could lead to some overlapping or cut-off issues in other designs. In some applications I have seen the indicator placed inside the button (marked with the red dots in the screenshot).

Reply:
yeah, also not super happy with how it looks!

cc @mattherzog for something more polished

Reply:
<img width="199" alt="image" src="https://github.com/livekit/components-js/assets/12997837/dd13645d-0569-4da9-93c0-9b502e481401">


^ what this branch shows now, but would be nicer if it was more polished

Reply:
yes!! 🔥